deployment/docker-compose/template.env

```diff
@@ -46,3 +46,8 @@ PROMETHEUS_MULTIPROC_DIR="/tmp"
 #### Whatsapp variables
 # WHATSAPP_VERIFY_TOKEN=
 # WHATSAPP_TOKEN=
+
+#### Urgency detection variables
+URGENCY_CLASSIFIER="llm_entailment_classifier"
```

Comment:
I don't like the idea of having two places to maintain whenever you add new classifier.

Reply:
And I am hoping we can add a bunch more easily

Reply:
I think we often get confused about env vars (especially others who might have not seen the PR but want to run something next time) since we don't document them very well...  Could we please take on this small overhead for now for the sake of less developer time wasted in the future?

If we do add a bunch we can add section to docs that explains it in full and we'll all be familiar with it by then too.